test/e2e/e2e.bats

```diff
@@ -2,25 +2,82 @@
 
 source test/e2e/helpers.sh
 
+setup() {
+	load 'helpers/bats-support/load'
+	load 'helpers/bats-assert/load'
+}
+
+teardown() {
+	run kubectl delete builds.shipwright.io --all
+	run kubectl delete buildruns.shipwright.io --all
+}
+
 @test "shp binary can be executed" {
-	result="$(shp --help)"
-	[ ! -z "$result" ]
+	run shp --help
+	assert_success
 }
 
-@test "shp build create succesfully run with 1 argument" {
+@test "shp environment variable lifecycle" {
+	# generate random names for our build and buildrun
 	build_name=$(random_name)
-	result="$(shp build create ${build_name} --source-url=url --output-image=image)"
-        [ ! -z "$result" ]
-	result=$(kubectl get build.shipwright.io ${build_name})
-	[ ! -z "$result" ]
+	buildrun_name=$(random_name)
+
+	# create a Build with two environment variables
+	run shp build create ${build_name} --source-url=https://github.com/shipwright-io/sample-go --output-image=my-image --env=VAR_1=build-value-1 --env=VAR_2=build-value-2
```

Comment:
@otaviof @coreydaley - I have not tried it yet, but perhaps you all have.  With the kind/local registry stuff we are employing, is it correct to say that by default we do *NOT* need credentials to push images to it?

And hence why I don't see cred creation here?

Reply:
Yes, we can assume the registry won't have any authentication, since we are in complete control of the container registry in use after #43 👍🏼